packages/utils/src/instrument.ts

```diff
@@ -247,8 +247,8 @@ export function parseFetchArgs(fetchArgs: unknown[]): { method: string; url: str
 }
 
 /** JSDoc */
-function instrumentXHR(): void {
-  if (!('XMLHttpRequest' in WINDOW)) {
+export function instrumentXHR(): void {
+  if (!(WINDOW as any).XMLHttpRequest) {
```

Comment:
q: Why do we need the typecast to `any` here? Or IOW why do we need it here and not in `WINDOW.document` below?

Reply:
ignore this, just learned that it's not in `WINDOW`. Makes sense!